### PR TITLE
upgrade to ruby 3.1.6

### PR DESCRIPTION
**Pull request by Smullz622**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -76,6 +77,11 @@ jobs:
       SETTINGS__solr__username: catalog
       SETTINGS__solr__password: catalog
     steps:
+    # - run:
```
Comment (jlandiseigsti) [inline .circleci/config.yml:80]: Do you think we'll turn uncomment this at any point? If not, we could just remove it.

**Review by jlandiseigsti — DISMISSED** [review]
👍

File: .circleci/config.yml (hunk 1 above)
Comment (ajkiessl) [reply .circleci/config.yml:80]: I think we should remove this.

**Review by ajkiessl — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (Smullz622) [reply .circleci/config.yml:80]: Good catch guys. That was leftover from some troubleshooting

**Review by ajkiessl — APPROVED** [review]

**Pull request merged**